OpenRA.Mods.Common/Traits/World/ShroudRenderer.cs

```diff
@@ -278,7 +278,7 @@ void CacheMapBorderShroud()
 		{
 			// Cache the whole of the map border shroud ahead of time, since it never changes.
 			Func<MPos, bool> mapContains = map.Contains;
-			foreach (var uv in CellRegion.Expand(map.Cells, 1).MapCoords)
+			foreach (var uv in map.AllCells.MapCoords)
```

Comment:
No, those don't make sense here.  We need to regenerate all the cells to clear any shroud sprites inside the visible bounds (ruling out the first), and the TerrainSpriteLayer can't/doesn't need to store data outside the AllCells area (ruling out the second).  `AllCells` is guaranteed to be at least as large as the old `CellRegion.Expand(map.Cells, 1).MapCoords` because of the >= 1 cell cordon.